GetAccept-v2/app.js

```diff
@@ -1447,7 +1557,7 @@
                         var sso_url = 'https://app.getaccept.com/auth/sso/login?token=' + escape(accessToken) + '&entity_id=' + entityId + '&go=' + escape(docUrl);
                         lbs.common.executeVba("GetAccept.OpenGALink", sso_url);
                         done();
-                    }, 3000);
+                    }, 11*1000);
```

Comment:
consider variable to make it more readable. Why 11*1000?

Reply:
I've tested it, and got 11 seconds for importing document and open in getaccept with ready document.
Described  [here](https://app.shortcut.com/getaccept/story/24248/lime-crm-add-delay-for-open-in-getaccept-when-uploading-file)

Reply:
can't we call the variable? `sleepWhileImporting` Or something then?